sdk/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpansProcessor.java

```diff
@@ -79,50 +106,137 @@ public void forceFlush() {
     // Do nothing.
   }
 
-  /**
-   * Returns a new Builder for {@link SimpleSpansProcessor}.
-   *
-   * @param spanExporter the {@code SpanExporter} to where the Spans are pushed.
-   * @return a new {@link SimpleSpansProcessor}.
-   * @throws NullPointerException if the {@code spanExporter} is {@code null}.
-   */
-  public static Builder newBuilder(SpanExporter spanExporter) {
-    return new Builder(spanExporter);
-  }
-
   /** Builder class for {@link SimpleSpansProcessor}. */
-  public static final class Builder {
+  @Immutable
+  @AutoValue
+  public abstract static class Config {
 
-    private final SpanExporter spanExporter;
-    private boolean sampled = true;
+    private static final boolean DEFAULT_EXPORT_ONLY_SAMPLED = true;
 
-    private Builder(SpanExporter spanExporter) {
-      this.spanExporter = Objects.requireNonNull(spanExporter, "spanExporter");
-    }
+    public abstract boolean isExportOnlySampled();
 
     /**
-     * Set whether only sampled spans should be reported.
+     * Creates a {@link SimpleSpansProcessor.Config} object using the default configuration.
      *
-     * @param sampled report only sampled spans.
-     * @return this.
+     * @return The {@link SimpleSpansProcessor.Config} object.
+     * @since 0.4.0
      */
-    public Builder reportOnlySampled(boolean sampled) {
-      this.sampled = sampled;
-      return this;
+    public static Config getDefault() {
+      return newBuilder().build();
     }
 
-    // TODO: Add metrics for total exported spans.
-    // TODO: Consider to add support for constant Attributes and/or Resource.
```

Comment:
Can you please leave these todos? Or create issue if you want.

Reply:
Could you please elaborate more on the second TODO? So I can provide a comprehensive description for the issue (and take it :wink:)